src/locales/ru/messages.po

```diff
@@ -302,73 +395,33 @@ msgid "ERC20"
 msgstr ""
 
 #: src/components/Landing/index.tsx
-msgid "Build a community around a project, fund it, and program its spending. Light enough for a group of friends, powerful enough for a global network of anons."
-msgstr "围绕一个项目建立一个社区，为其提供资金，并对其支出进行规划。既能简单易用让一群知己好友轻松创建属于自己的社区项目，也可以强大到能够支持一个全球匿名网络的运作"
```

Comment:
@peripheralist might be good to merge this MR ASAP, it looks like some Chinese translations snuck their way into the Russian translation file (note: I suspect this was due to merge conflict resolution, not a result of Crowdin)